composer.json

```diff
@@ -35,7 +35,7 @@
         "drupal/coder": "^8.3.1",
         "drush/drush": "^9.5.0",
         "drupal/config_split": "^1.0.0",
-        "drupal/core": "^8.6.0",
+        "drupal/core": "^8.5.0",
```

Comment:
Do we really want this? Drupal 8.7.0 is almost out of the door, which will de facto make 8.5.x deprecated entirely. What we can refrain from doing, however, is to update core to `^8.7.0` and leave it to `^8.6.0`.

Reply:
Well, @anavarre, I'm ready to roll out ORCA functionality that automatically selects the two supported releases via Composer and tests against them. I could roll it out today if it weren't for this line. I would prefer to make this change today and revert it a short time in the future. It's a trivial effort, and we get significant instant value from it.

Reply:
...oh, and it should have no impact whatsoever on current users. Even new users will automatically get the latest version of Drupal Core by virtue of Composer's logic. To oversimplify things, this helps ORCA and has no effect on anyone else. 😃